src/main/java/com/targomo/client/api/pojo/TravelWeight.java

```diff
@@ -0,0 +1,19 @@
+package com.targomo.client.api.pojo;
+
+/**
+ * @author Mario Freitas
+ */
+
+import lombok.AllArgsConstructor;
+
+/**
+ * This class is responsible for storing the distance and time taken to route between a source and target
+ * @author Mario Freitas
+ */
+@AllArgsConstructor
+public class TravelWeight {
+
+    public final int travelDistance;
```

Comment:
I think I would prefer having these private and add @Getter, @Setter or is there a reason to have it public?